src/configurationMenu.ts

```diff
@@ -181,13 +181,9 @@ export class ConfigurationMenu {
 
 	async configure() {
 		await this.collectInputs();
-		['accessToken', 'project', 'env'].forEach(async option => {
+		['project', 'env'].forEach(async option => {
```

Comment:
is configure no longer asking for Access Token when going through the flow? Not sure exactly what this is changing

Reply:
No, it still asking for `accessToken`, but this piece of code is where it tries to update the old accessToken config.